internal/runners/config/set.go

```diff
@@ -52,6 +55,14 @@ func (s *Set) Run(params SetParams) error {
 		if err != nil {
 			return locale.WrapInputError(err, "Invalid integer value")
 		}
+	case configMediator.Enum:
+		enums := option.Default.(*configMediator.Enums)
```

Comment:
I believe this is a potential panic if the type check fails. We should add an `ok` check here.

Reply:
I had more faith in us, but very well, I've added a check :)